packages/sdks-tests/src/helpers/visual-editor.ts

```diff
@@ -73,6 +73,32 @@ export const sendContentUpdateMessage = async ({
   );
 };
 
+export const sendNewStateMessage = async ({
+  page,
+  newState,
```

Comment:
we need to pass `model` inside data as we're checking for it in the gen1 SDK:

https://github.com/BuilderIO/builder/blob/f9b85528015839287fbf092470bc57c38344d329/packages/react/src/components/builder-component.component.tsx#L549-L551

Reply:
Oh boy look at that, all tests passed 😍 
Thanks again! 👍🏽